### DISCUSSION TITLE
Best Crazyflie Simulator

### DISCUSSION THREAD

**Discussion by sesem738** [bitcraze/discussions#1390] [Q&A]
I have seen a couple of community created simulators for the Crazyflie. Which one does the best job at emulating the Crazyflie's firmware and python library?

**Comment by knmcguire** [comment]
Which ones are you looking at in particular? perhaps I can get the creators to pitch in on this since this is a bit of a tough question for us.

**Reply by sesem738** [reply]
I have settled on CrazySim. I just needed something that I can do a 1-to-1 sim-to-real transfer in terms of the Crazyflie's Python API

**Reply by knmcguire** [reply]
Crazysim would be indeed a good choice:) There is also sim_cf2 that you can look at. It uses gazebo classic though so that is perhaps also something to consider https://www.bitcraze.io/2024/01/simulation-sim_cf2-for-multi-agent-algorithm-development/
